app/views/shared/_user_show_fields.html.slim

```diff
@@ -0,0 +1,12 @@
+article.user-information class="user-id-#{locals[:user].id}"
+  h2 Pirate Info
+  p.user_id User ID: #{locals[:user].id}
+  p.user_name User Name: #{locals[:user].name}
+  p.user_tone_name Tone Name: #{(locals[:user].tone_name) ? locals[:user].tone_name : "(none)"}
+  p.user_email Email Address: #{locals[:user].email}
+  p.user_start_date Motley Tone since: #{locals[:user].band_start_date.strftime("%d-%b-%Y")}
+  - if locals[:user].admin?
```

Comment:
That is the intended behavior.  A non-admin pirate needs to know who the admin pirates are when admin tasks need to be done for them.